common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -55,7 +55,18 @@ trait ABTestSwitches {
     "Test the impact of showing the user a component that highlights the Guardians journalism",
     owners = Seq(Owner.withEmail("dotcom.platform@guardian.co.uk")),
     safeState = Off,
-    sellByDate = Some(LocalDate.of(2024, 12, 18)),
+    sellByDate = Some(LocalDate.of(2025, 1, 29)),
```

Comment:
Was extending this other switch intended to be part of this change?

Reply:
Yeah, I don't want this footgun catching us out just before christmas